optas/models.py

```diff
@@ -1827,3 +1827,142 @@ def get_global_link_axis_function(
         return self.make_function(
             "a", link, get_global_link_axis, n=n, numpy_output=numpy_output
         )
+    
+    def computeRNEA(
+            self,
+            q: ArrayType, 
+            qd: ArrayType, 
+            qdd: ArrayType, 
+            extf: ArrayType,
+            extTau: ArrayType
```

Comment:
These are not necessary. You should remove so that rnea computes: `tau = RNEA(q, dq, ddq)`

Reply:
This part is removed